### PR TITLE
Improve wording on Readme

### PR DESCRIPTION
**Pull request by RichardPoes**
Fixes https://github.com/MarcDirven/cpp-lazy/issues/67

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,10 +5,10 @@
...
+Cpp-lazy is a fast and easy lazy evaluation library for C++11/14/17/20. This is a fast library is because the library doesn't allocate any memory. Moreover, the iterators are random-access where possible. Therefore operations, for example `std::distance`, are an O(1) operation, either by "overloading" the `std::distance`/`std::next` functions using ADL lookup, or by adding a `std::random_access_iterator_tag`. Furthermore, the view object has many `std::execution::*` overloads. This library uses one (optional) dependency: the library `{fmt}`, more of which can be found out in the [installation section](https://github.com/MarcDirven/cpp-lazy#Installation).
```
Comment (MarcDirven) [inline README.md:8]: "This is a fast library is because"

File: README.md [hunk 2]
```diff
@@ -23,18 +23,14 @@ int main() {
 - Compatible with old(er) compiler versions; at least `gcc` versions => `4.8` & `clang` => `5.0.0` (previous 
 versions have not been checked, so I'd say at least a compiler with C++11 support).
 - Tested with `-Wpedantic -Wextra -Wall -Wno-unused-function` and `/W4` for MSVC
-- One dependency ([`fmt`](https://github.com/fmtlib/fmt)) which is automatically configured, or none if you use `#define LZ_STANDALONE` before including or use CMake `-D CPP-LAZY_USE_STANDALONE`
+- One dependency ([`{fmt}`](https://github.com/fmtlib/fmt)) which is automatically configured, or none if you use `#define LZ_STANDALONE` before including or use CMake `-D CPP-LAZY_USE_STANDALONE`
```
Comment (MarcDirven) [inline README.md:26]: Refer to installation maybe

**Review by MarcDirven — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (RichardPoes) [reply README.md:26]: I do not agree since this is just a list of features. It is already refered to before when `{fmt}` is mentioned for the first time.

Comment (MarcDirven) [reply README.md:26]: Then "One dependency `{fmt}`" will suffice

**Review by MarcDirven — APPROVED** [review]

**Pull request merged**